docs/execution-providers/QNN-ExecutionProvider.md

```diff
@@ -52,10 +52,18 @@ OnnxRuntime QNN Execution Provider is a supported runtime in [Qualcomm AI Hub](h
 ## Configuration Options
 The QNN Execution Provider supports a number of configuration options. These provider options are specified as key-value string pairs.
 
+|`"backend_type"`|Description|
+|---|-----|
+|'cpu'|Enable CPU backend. Useful for integration testing. The CPU backend is a reference implementation of QNN operators.|
+|'htp'|Enable HTP backend. Offloads compute to NPU. Default.|
```

Comment:
we should list all supported backend_type like gpu and saver

Reply:
ok. I added those too, but I don't know what else to say about them. let me know if there are details I should include.